tasks/main.yml

```diff
@@ -1,23 +1,26 @@
 ---
-- name: Configure the Docker APT key
-  apt_key: keyserver="{{ docker_keyserver }}"
-           id="58118E89F3A912897C070ADBF76221572C52609D"
-           state=present
+- name: Check python version
```

Comment:
It is frustrating that these steps are necessary to make things work is a secure way.

I think that we need to look at installing the necessary dependencies outside of this role. Doing it at the project level would work, but would likely become repetitive. My initial instincts are that we should create a new Ansible role called `python-security`, or something like that. It should install all dependencies necessary to make Python 2.7.6 HTTPS connections secure.